week4/assignment/week4_assignment/src/pages/MyPage/FindHobby.tsx

```diff
@@ -0,0 +1,115 @@
+import { useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { checkingInstance } from '../../api/axios';
+import { Theme } from '../../styles/Themes';
+
+const FindHobby = () => {
+  const [no, setNo] = useState<number | null>(null);
+  const [hobby, setHobby] = useState('');
+  const [othersHobby, setOthersHobby] = useState('');
+  const [selectedHobby, setSelectedHobby] = useState('');
+
+  useEffect(() => {
+    setSelectedHobby(`${no}번 사용자의 취미: ${othersHobby}`);
+  }, [othersHobby]);
+
+  const getMyHobby = async () => {
+    try {
+      const response = await checkingInstance.get('/user/my-hobby');
+      return response.data.result.hobby;
+    } catch (e) {
+      console.log(e);
```

Comment:
```suggestion
      console.error(e);
```

에러메시지의 경우 console.log보다는 console.error로 찍어주면 더 좋을 것 같아요~